lesson5/index.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Recap Functions & Methods
+nav_order: 5
+has_children: true
+---
+
+# Lesson 4: Recap Functions & Methods
+
+## Goals
+* Recap method `signature`, `arguments` and `return` value
+* Recap `recursion`
```

Comment:
We should add a link to this: https://www.google.com/search?source=hp&ei=m3ZXX8CVHMrWatmLg6gJ&q=recursion&oq=recursion&gs_lcp=CgZwc3ktYWIQAzIFCAAQsQMyAggAMgIIADICCAAyAggAMgIIADICCAAyAggAMgIIADICCAA6CAgAELEDEIMBOgUILhCxAzoKCAAQsQMQRhD_AVC_B1iCEWCJE2gBcAB4AIABU4gBxgSSAQE5mAEAoAEBqgEHZ3dzLXdperABAA&sclient=psy-ab&ved=0ahUKEwiAz7asxdnrAhVKqxoKHdnFAJUQ4dUDCAk&uact=5 :-)

Reply:
I added it to the header of the recursion section.